docs/LICENSE

```diff
@@ -0,0 +1,18 @@
+Copyright 2020 Anita-B
```

Comment:
out of curiosity, why is this license here? is this needed? isn't the project license enough 🤔

Reply:
Yes, I will remove it.